generated/nidcpower/nidcpower/lcr_load_compensation_spot.py

```diff
@@ -0,0 +1,118 @@
+import ctypes
+import numbers
+import platform
+
+import nidcpower._visatype
+import nidcpower.enums as enums
+
+
+# This class is an internal ctypes implementation detail that corresponds to
+# NILCRLoadCompensationSpot in the C API
+class struct_NILCRLoadCompensationSpot(ctypes.Structure):  # noqa N801
+    if platform.system() == "Windows" and platform.architecture()[0] == "64bit":
+        _pack_ = 8
+    elif platform.system() == "Linux" or platform.system() == "Darwin":
+        _pack_ = 4
+    else:
+        _pack_ = 1
+    _fields_ = [
+        ("frequency", nidcpower._visatype.ViReal64),
+        ("reference_value_type", nidcpower._visatype.ViInt32),
+        ("reference_value_a", nidcpower._visatype.ViReal64),
+        ("reference_value_b", nidcpower._visatype.ViReal64),
+    ]
+
+    def __init__(self, data):
+        super(ctypes.Structure, self).__init__()
+        self.frequency = data.frequency
+        self.reference_value_type = enums.LCRReferenceValueType(data.reference_value_type).value
+        if self.reference_value_type == enums.LCRReferenceValueType.IMPEDANCE.value:
+            self.reference_value_a = data.reference_value.real
+            self.reference_value_b = data.reference_value.imag
+        else:
+            self.reference_value_a = data.reference_value
+            self.reference_value_b = 0.0
+
+
+class LCRLoadCompensationSpot(object):
+    """Constructs a DUT specification for a given frequency to use in LCR load compensation."""
+
+    def __init__(
+        self,
+        frequency=0.0,
+        impedance=None,
+        ideal_capacitance=None,
+        ideal_inductance=None,
+        ideal_resistance=None,
+    ):
+        """LCRLoadCompensationSpot
+
+        Creates and returns an LCRLoadCompensationSpot object. At most one of impedance,
+            ideal_capacitance, ideal_inductance and ideal_resistance can be set, and the remaining
+            parameters (excluding frequency) must be None. The parameter that is not None specifies
+            the known specification value of the DUT to be used as the basis for load compensation.
+            If all of them are None, then the default value of `impedance=complex()` will be used.
```

Comment:
I have updated this docstring based on your suggestion in my latest commit. Note that I've removed the statement related to default value since the code no longer sets a default value when all the parameters are None.